juliac.jl

```diff
@@ -42,6 +42,10 @@ function main(args)
         "--clean", "-c"
             help = "delete builddir"
             action = :store_true
+        "--target", "-t"
+            help = "compilations target"
+            arg_type = String
+            default = "x86-64"
```

Comment:
default value for `-march=` should be "generic" (or the empty string)

Reply:
I used `"x86-64"` because that is the value returned in the command `Base.julia_cmd()`